Source/RageCommunity.Library/Wrappers/NativeVehicleWrappers.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Rage;
+using Rage.Native;
+
+namespace RageCommunity.Library.Wrappers
+{
+    public static partial class NativeWrappers
+    {
+        /// <summary>
+        /// Causes the given vehicle to honk it's horn for a specified duration in seconds.
+        /// </summary>
+        /// /// <param name="mode">"NORMAL" or "HELDDOWN"</param>
```

Comment:
To much /// ;) 
We could also think about making this a boolean.

Reply:
Resolved in new commits